.papr.yml

```diff
@@ -41,6 +41,7 @@ extra-repos:
 # XXX: integrate better with ci/ scripts
 tests:
   - yum-builddep -y rpm-ostree
+  - yum install -y polkit-devel
```

Comment:
Ahh right, we'll also need this at least temporarily added to `install_builddeps` in `libbuild.sh`. Hopefully we can eventually make the spec file in this repo be the canonical one to make this easier.